src/uct/api/v2/uct_v2.h

```diff
@@ -599,11 +599,10 @@ typedef struct uct_iface_is_reachable_params {
 /**
  * @ingroup UCT_RESOURCE
  * @brief Operation parameters passed to @ref uct_ep_is_connected.
- *        Only valid addresses which are present in remote_address should be
- *        passed.
  *
  * This struct is used to pass the required arguments to
- * @ref uct_ep_is_connected.
+ * @ref uct_ep_is_connected. Only valid addresses which are present in
+ * remote_address should be passed.
```

Comment:
what is remote_address ? does it refer to some field?

Reply:
actually maybe it's not relevant anymore because of the details for each field.